samples_project/Assets/SampleViewer/Samples/Measure/Measure.cs

```diff
@@ -101,28 +103,31 @@ private void Interpolate(GameObject start, GameObject end, List<GameObject> feat
         double d = ArcGISGeometryEngine.DistanceGeodetic(startPoint, endPoint, currentUnit, new ArcGISAngularUnit(ArcGISAngularUnitId.Degrees), ArcGISGeodeticCurveType.Geodesic).Distance;
         float n = Mathf.Floor((float)d / InterpolationInterval);
         double dx = (end.transform.position.x - start.transform.position.x) / n;
+        double dy = (end.transform.position.y - start.transform.position.y) / n;
         double dz = (end.transform.position.z - start.transform.position.z) / n;
 
-        var pre = start.transform.position;
+        var PreviousInterpolation = start.transform.position;
```

Comment:
```suggestion
        var previousInterpolation = start.transform.position;
```
Should be camelcase for local variable in C#.